File: jupyter2/problems/iteration/series_sum/ml/series_sum.ml
(** begin my answer *)

(** end my answer *)

let main () =
  let a = Float.pi *. Float.pi /. 6.0 in
  (assert (abs_float (series_sum 10 -. 1.549768) < 1.0e-5);
   assert (abs_float (series_sum 100000 -. a) < 1.0e-5);
   assert (abs_float (series_sum 1000000 -. a) < 1.0e-6);
   assert (abs_float (series_sum 20000000 -. a) < 1.0e-6));
  Printf.printf "OK\n"
;;

main()

File: jupyter2/problems/iteration/series_sum/go/series_sum.go
package main
import "math"
import "fmt"

/** begin my answer */

/** end my answer */

func main() {
	a := math.Pi * math.Pi / 6.0
	if !(math.Abs(seriesSum(10) - 1.549768) < 1.0e-5) { panic("wrong") }
	if !(math.Abs(seriesSum(100000) - a) < 1.0e-5) { panic("wrong") }
	if !(math.Abs(seriesSum(1000000) - a) < 1.0e-6) { panic("wrong") }
	if !(math.Abs(seriesSum(20000000) - a) < 1.0e-6) { panic("wrong") }
	fmt.Println("OK")
}
